runtime/src/bank/executor_cache.rs

```diff
@@ -0,0 +1,377 @@
+use std::{
```

Comment:
Oh, also I think it would be helpful to add some `//!` doc comments to the new files. Even if it's basic, this is where I look first to see if I can learn about intentions and interactions of the contained code.